src/app.py

```diff
@@ -0,0 +1,101 @@
+from flask import Flask, request, jsonify, session
+import json_connection as data_connection
+
+app = Flask(__name__)
+app.secret_key = "jmjs7"
```

Comment:
we shouldn't save secrets in source files, please use .env file for that

https://lukianovihor.medium.com/python-environment-variables-using-dotenv-library-71529ad0e9c3